src/demo/components/contacts/ContactsRow.tsx

```diff
@@ -40,7 +40,7 @@ export function ContactsRow(props: ContactRowProps): React.ReactElement {
 
     const getContext = (): ContactContext => {
         return {
-            type: "contact",
+            type: 'fdc3.contact',
```

Comment:
I was wondering if we should get rid of the `Intents` enum. The intents defined in the service are only the "built-in" intents, with the expectation that applications will add many more intent types. Since an enum can't be extended, it's actual usefulness to a user of the service may be limited.

But yeah, should definitely be consistent with intents - just a question of which way that should go, should discuss.